### PR TITLE
chore: refactor Iota2.0 consensus ChainOutputs

### PR DESCRIPTION
**Pull request by lmoe**

### CODE REVIEW

File: packages/chain/cons/cons.go [hunk 1]
```diff
@@ -553,12 +554,13 @@ func (c *consImpl) uponVMInputsReceived(aggregatedProposals *bp.AggregatedBatchP
...
-		AnchorOutput:         decidedBaseAnchorOutput.GetAnchorOutput(),
-		AnchorOutputID:       decidedBaseAnchorOutput.OutputID(),
-		Store:                c.chainStore,
-		Requests:             aggregatedProposals.OrderedRequests(requests, *randomness),
-		TimeAssumption:       aggregatedProposals.AggregatedTime(),
+		Processors: c.processorCache,
+		Inputs:     decidedBaseAnchorOutput,
+		Store:      c.chainStore,
+		Requests:   aggregatedProposals.OrderedRequests(requests, *randomness),
+		// TODO: <lmoe> Is TimeAssumption->Timestamp a 1:1 change?
```
Comment (jorgemmsilva) [inline packages/chain/cons/cons.go:561]: think so, ye

File: packages/chain/cons/cons.go [hunk 2]
```diff
@@ -611,34 +614,44 @@ func (c *consImpl) uponVMOutputReceived(vmResult *vm.VMTaskResult) gpa.OutMessag
...
-		Essence: resultTxEssence,
-		Unlocks: transaction.MakeSignatureAndAliasUnlockFeatures(len(resultTxEssence.Inputs), signatureForUnlock),
+
+	resultInputs, err := resultTx.Inputs()
+	if err != nil {
+		panic(fmt.Errorf("cannot get inputs from result TX: %w", err))
 	}
+
+	// TODO: <lmoe> This is most likely just trash :D
+	tx := &iotago.SignedTransaction{
```
Comment (jorgemmsilva) [inline packages/chain/cons/cons.go:632]: why do we create a `SignedTransaction` here?

Comment (lmoe) [reply packages/chain/cons/cons.go:632]: Late night uneducated guess, based on the model. It seemed to be the only way to set the Unlocks. 
I added the comment to take a later look again.

Comment (jorgemmsilva) [reply packages/chain/cons/cons.go:632]: yeah, I think you're right :+1: , this seems correct

**Review by jorgemmsilva — APPROVED** [review]

File: packages/chain/cons/cons.go (hunk 1 above)
Comment (dessaya) [reply packages/chain/cons/cons.go:561]: Yes, just a rename

**Pull request merged**